www/views/page/content/list.php

```diff
@@ -16,9 +16,7 @@
 $catFilter = Session::instance()->getFilter();
 $filtercat = isset($_GET['filtercat']) ? intval($_GET['filtercat']) : 0;
 $categories = Category::getAll();
-if (sizeof($catFilter) == sizeof($categories)) {
-    $catFilter = [];
-}
+$showFilters = isset($_GET['show_filter_options']) ? 1 : 0;
```

Comment:
Volgens mij kan dit gewoon met

```suggestion
$showFilters = isset($_GET['show_filter_options'])
```